files-override/shared/.prettierrc.cjs

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+module.exports = {
```

Comment:
again this is a new file, can we split this into a new PR?

Reply:
I'll have to do the shared/js split first, ya